<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMessage</class>
 <widget class="QMainWindow" name="ClientMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Message</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditUsername">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelContact">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddContact">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMessage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Контакты</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>331</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>159</width>
        <height>329</height>
       </rect>
      </property>
      <widget class="QListWidget" name="listWidgetContacts">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>161</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Сообщение</string>
    </property>
    <widget class="QListWidget" name="listWidgetMessages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearTextButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отключение</string>
    </property>
   </widget>
   <zorder>pushButtonAddContact</zorder>
   <zorder>textEditUsername</zorder>
   <zorder>pushButtonDelContact</zorder>
   <zorder>textEditMessage</zorder>
   <zorder>pushButtonSend</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>clearTextButton</zorder>
   <zorder>connectButton</zorder>
   <zorder>disconnectButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
